adagucserverEC/CGeoJSONData.h

```diff
@@ -108,7 +108,21 @@ class FeatureProperty {
       s.print("NONE");
     }
     return s;
-  }
+  };
+
+  CT::string toString(const char *fmt) {
```

Comment:
Can you keep the declaration here and move the implementation to the CPP file?

Reply:
Yep